astroquery/esa_hubble/tests/test_esa_hubble.py

```diff
@@ -0,0 +1,142 @@
+from astropy import coordinates
+# from astropy.tests.helper import remote_data
+
+
+class TestESAHubble():
+    
+    def get_dummy_tap_handler(self):
+        parameterst = {}
+        parameterst['query'] = "select top 10 * from hsc_v2.hubble_sc2"
+        parameterst['output_file'] = "test2.vot"
+        parameterst['output_format'] = "votable"
+        parameterst['verbose'] = False
+        dummyTapHandler = DummyESAHubbleTapHandler("launch_job", parameterst)
+        return dummyTapHandler
+
+    def test_get_product(self):
+        parameters = {}
+        parameters['observation_id'] = "J6FL25S4Q"
+        parameters['calibration_level'] = "RAW"
+        parameters['verbose'] = False
+        dummyHandler = DummyHandler("get_product", parameters)
+        ehst = ESAHubbleClass(dummyHandler, self.get_dummy_tap_handler())
+        ehst.get_product("J6FL25S4Q", "RAW")
+        dummyHandler.check_call("get_product", parameters)
+
+    def test_get_artifact(self):
+        parameters = {}
+        parameters['artifact_id'] = "O5HKAX030_FLT.FITS"
+        parameters['verbose'] = False
+        dummyHandler = DummyHandler("get_artifact", parameters)
+        ehst = ESAHubbleClass(dummyHandler, self.get_dummy_tap_handler())
+        ehst.get_artifact("O5HKAX030_FLT.FITS")
+        dummyHandler.check_call("get_artifact", parameters)
+
+    def test_get_postcard(self):
+        parameters = {}
+        parameters['observation_id'] = "X0MC5101T"
+        parameters['verbose'] = False
+        dummyHandler = DummyHandler("get_postcard", parameters)
+        ehst = ESAHubbleClass(dummyHandler, self.get_dummy_tap_handler())
+        ehst.get_product("X0MC5101T")
+        dummyHandler.check_call("get_postcard", parameters)
+
+    def test_query_target(self):
+        parameters = {}
+        parameters['name'] = "m31"
+        parameters['verbose'] = False
+        dummyHandler = DummyHandler("query_target", parameters)
+        ehst = ESAHubbleClass(dummyHandler, self.get_dummy_tap_handler())
+        ehst.query_target(parameters['name'])
+        dummyHandler.check_call("query_target", parameters)
+
+    def test_cone_search(self):
+        c = coordinates.SkyCoord("00h42m44.51s +41d16m08.45s", frame='icrs')
+        parameterst = {}
+        parameterst['query'] = "select top 10 * from hsc_v2.hubble_sc2"
+        parameterst['output_file'] = "test2.vot"
+        parameterst['output_format'] = "votable"
+        parameterst['verbose'] = False
+        dummyTapHandler = DummyESAHubbleTapHandler("launch_job", parameterst)
+
+        parameters = {}
+        parameters['coordinates'] = c
+        parameters['radius'] = None
+        parameters['file_name'] = None
+        parameters['verbose'] = False
+        dummyHandler = DummyHandler("cone_search", parameters)
+
+        ehst = ESAHubbleClass(dummyHandler, dummyTapHandler)
+        ehst.cone_search(parameters['coordinates'])
```

Comment:
This line is triggering an error:
`E               urllib.error.URLError: <urlopen error An attempt was made to connect to the internet by a test that was not marked `remote_data`. The requested host was: archives.esac.esa.int>`
which is related to the remote_data issue I reported